spec/lib/bandiera/client_spec.rb

```diff
@@ -4,7 +4,41 @@
   let(:base_uri)  { 'http://bandiera.com' }
   let(:api_uri)   { "#{base_uri}/api" }
   let(:logger)    { double.as_null_object }
-  subject         { Bandiera::Client.new(api_uri, logger) }
+  let(:client_instance) { Bandiera::Client.new(api_uri, logger) }
```

Comment:
Actually, the way the spec refactoring turned out, I didn't need it.  I'll go back to subject.